### ISSUE TITLE
Twig Block Short Tag Duplicated

### ISSUE THREAD

**Issue by wolfmoritz** [mblode/vscode-twig-language-2#39]
In one of my templates I have an empty block short tag as a placeholder, to be extended in other templates:

```
  {% block head '' %}
```
After formatting, this tag is duplicated causing compile errors:
```
            {% block head '' %}
        {% block head '' %}
```
I logged this under Twig Language, as I found the issue in both extensions mblode/vscode-twig-language#30

**Comment by mblode** [comment]
This has been fixed in the latest update 👍

**Comment by mblode** [comment]
Fixed block shorthand formatting in 0.12.0 and added a regression test.
